src/main/java/org/openstreetmap/atlas/checks/flag/CheckFlag.java

```diff
@@ -105,11 +109,65 @@ public CheckFlag(final String identifier, final Set<? extends AtlasObject> objec
      */
     public CheckFlag(final String identifier, final Set<? extends AtlasObject> objects,
             final List<String> instructions, final List<Location> points)
+    {
+        this(identifier, objects, instructions, points, new HashSet<>());
+    }
+
+    /**
+     * Creates a {@link CheckFlag} with the addition of a list of {@code point} {@link Location}s
+     * that highlight specific points in the geometry that caused the rule violation
+     *
+     * @param identifier
+     *            the identifying value to flag
+     * @param objects
+     *            {@link AtlasObject}s to flag
+     * @param instructions
+     *            a list of free form instructions
+     * @param points
+     *            {@code point} {@link Location}s to highlight
+     * @param fixSuggestions
+     *            {@link Set} of {@link FeatureChange}s representing suggested fixes for flagged
+     *            features
+     */
+    public CheckFlag(final String identifier, final Set<? extends AtlasObject> objects,
```

Comment:
I wanted to avoid adding constructors and instead use setters for this, as we already have a lot of CheckFlag constructors and methods that call those constructors in BaseCheck. I had to add this constructor, though, to maintain the serializability of the class.